pkg/deployments/tasks/20210418-vault/input.ts

```diff
@@ -0,0 +1,17 @@
+import Task from '../../src/task';
+
+export type VaultDeployment = {
+  authorizer: string;
+  weth: string;
+  pauseWindowDuration: number;
+  bufferPeriodDuration: number;
+};
+
+const authorizer = new Task('20210418-authorizer');
+
+export default {
+  authorizer,
+  weth: '0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2',
```

Comment:
I'm still understanding how this is structured, but note that this address is only valid for mainnet.